src/transformState.js

```diff
@@ -6,6 +6,35 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    switch (true) {
+      case action.type === 'addProperties': {
```

Comment:
That seems the same as using `if`
write `action.type` in `switch` instead of true